wgsl/index.bs

```diff
@@ -520,6 +520,76 @@ type.
 
 ## Texture Types TODO ## {#texture-types}
 
+A <dfn noexport>texel</dfn> is a scalar or vector used as the smallest independently addressible element of a [=texture=].
+The word *texel* is short for *texture element*.
+
+A <dfn noexport>texture</dfn> is a grid of texels supporting special operations useful for rendering.
+A texture's representation is typically optimized for those operations, with many details hidden from the programmer
```

Comment:
it's a bit weird to call them "special operations useful for rendering" and then follow with "those operations", which assumes the users know what they are. Would be good to provide examples here

Reply:
I wanted to defer those to the details of builtin functions.  That's why later in this introductory section we get into the fact there's no direct access to texture storage and that everything is determined by the set of available texture builtins.

Maybe I can think of something that points forward to the builtins without going into details.

Frankly, I'm concerned about having to describe sampling operations. I'd rather not do that in the WGSL spec.  Like, there's a *lot* of content in Vulkan's "14. Image operations". https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#textures